### ISSUE TITLE
Should org-caldav-sync save the org file after inserting the properties section?

### ISSUE THREAD

**Issue by mhcerri** [dengste/org-caldav#148]
I noticed that `org-caldav-sync` leaves the calendar org file open and unsaved after it inserts the property section.

Is that the expected behavior? And if so, is there a way to force it to save the file and close its buffer?

**Comment by dengste** [comment]
Yes, this is expected. There is currently no way to save automatically, but shouldn't be hard to add.
